src/components/landscape-sidebar/geo-description-widget/geo-description-widget.js

```diff
@@ -1,7 +1,19 @@
+import React from 'react';
 import { connect } from 'react-redux';
 import Component from './geo-description-widget-component';
 import mapStateToProps from './geo-description-widget-selectors';
+
 // this forces the registration of redux module and sagas
 import 'redux_modules/geo-description';
 
-export default connect(mapStateToProps, null)(Component);
+const GeoDescriptionWidget = props => {
+  const handleBackClick = () => {
+    const { view } = props;
+    const target = { center: view.center, zoom: 7 }
```

Comment:
I've been testing the `goTo` function and it seems that the delay on the movement on the camera is due to the way we are passing the params to the function. It works fine by setting the `position`. Something like this:
```js
const handleBackClick = () => {
    const { view } = props;
    const { center } = view;
    const target = {
      position: {
        x: center.longitude,
        y: center.latitude,
        z: 1000000
      },
      heading: 0,
      tilt: 0
    }
    view.goTo(target)
  };
```